src/group_theory/submonoid/operations.lean

```diff
@@ -388,25 +388,29 @@ end galois_insertion
 
 end submonoid
 
-namespace submonoid_class
+namespace one_mem_class
```

Comment:
It's out of scope for this PR, but I'd argue that all of these belong in the `set_like` namespace. That would make a good follow-up PR

Reply:
Yes, I think it would be good to put virtually nothing in the various `subobject_class` namespaces and dump everything into `set_like`. I think there should be very few naming conflicts.